tests/test_api_build.py

```diff
@@ -412,6 +412,9 @@ def dummy_executable(folder, exename):
     return exename
 
 
+@pytest.mark.skip(
+    reason="GitHub discontinued SVN, see https://github.com/conda/conda-build/issues/5098"
+)
```

Comment:
GitHub has started the processes of turning off SVN support causing this tests to fail. Refactoring revealed unexpected surprises, so I've chosen to step back for this hotfix and temporarily bypass the failing test.

See https://github.com/conda/conda-build/issues/5098 for more.